src/utils/requests/__mocks__/BillRequests.js

```diff
@@ -0,0 +1,38 @@
+const mockGetBill = jest.fn((token) => {
```

Comment:
This isn't used in our codebase, so it's not exactly what I meant by mocking. I wrote it above but you mock the fetch api call so that way you can test the getBill function you created inside the BillRequests file.